df.world.xml

```diff
@@ -703,6 +703,20 @@
         <int32_t/>
     </struct-type>
 
+    <struct-type type-name='divination_set_unk_v47_5_1'>
+        <stl-vector name='index' type-name='int32_t' comment="weird implementation, so there's probably something else to it"/>
+        <int32_t name='die' comment="Looks like enum 0-4 for D20, D12, D8, D6, and D4, but what's the order"/>
+        <int32_t name='unk_v47_2' comment="Effect? Wide range of numbers, 0 to over 10000"/>
+    </struct-type>
+    
+    <struct-type type-name='divination_set'>
+        <int32_t name='index' comment="into vector"/>
```

Comment:
Are you sure this isn't an `id` instead? A lot of similar vectors are sorted by ID, and if items haven't ever been removed, IDs will correspond directly to indices.

Reply:
You're right, of course.